CMakeLists.txt

```diff
@@ -168,6 +168,9 @@ option(BUILD_LIBRARIES_ONLY "BUILD_LIBRARIES_ONLY" OFF)
 # build the GUI component, when disabled only nextcloudcmd is built
 option(BUILD_GUI "BUILD_GUI" ON)
 
+# build the tests
+option(BUILD_TESTING "BUILD_TESTING" ON)
```

Comment:
why is this needed?

Reply:
to get proper usage of the option within cmake build system
like for example being able to turn it on/off from Qt Creator